src/stability_sdk/utils.py

```diff
@@ -1,23 +1,63 @@
-import pathlib
-import sys
-import os
-import uuid
-import random
 import io
 import logging
-import time
-from typing import Dict, Generator, List, Optional, Union, Any, Sequence, Tuple
-import mimetypes
-
+import os
 
 from PIL import Image
+from typing import Dict, Generator, Optional, Sequence, Tuple, Type, TypeVar, Union
+
+from .api import generation
+from .matrix import Matrix
 
-import stability_sdk.interfaces.gooseai.generation.generation_pb2 as generation
-import stability_sdk.interfaces.gooseai.generation.generation_pb2_grpc as generation_grpc
 
 logger = logging.getLogger(__name__)
 logger.setLevel(level=logging.INFO)
 
+MAX_FILENAME_SZ = int(os.getenv("MAX_FILENAME_SZ", 200))
+
+
+#==============================================================================
+# Mappings from strings to protobuf enums
+#==============================================================================
+
+BORDER_MODES = {
+    'replicate': generation.BORDER_REPLICATE,
+    'reflect': generation.BORDER_REFLECT,
+    'wrap': generation.BORDER_WRAP,
+    'zero': generation.BORDER_ZERO,
+    'prefill': generation.BORDER_PREFILL,
+}
+
+CAMERA_TYPES = {
+    'perspective': generation.CAMERA_PERSPECTIVE,
+    'orthographic': generation.CAMERA_ORTHOGRAPHIC,
+}
+
+COLOR_MATCH_MODES = {
+    "hsv": generation.COLOR_MATCH_HSV,
+    "lab": generation.COLOR_MATCH_LAB,
+    "rgb": generation.COLOR_MATCH_RGB,
+}
+
+GUIDANCE_PRESETS: Dict[str, int] = {
+    "none": generation.GUIDANCE_PRESET_NONE,
+    "simple": generation.GUIDANCE_PRESET_SIMPLE,
+    "fastblue": generation.GUIDANCE_PRESET_FAST_BLUE,
```

Comment:
are there better names we could use for fastblue/fastgreen? this is at least consistent with the API docs, but i don't think these names ever appeared in the dreamstudio UI, and now they show up in the gradio animation UI. it's a small thing, but i wouldn't blame users for thinking these are new presets

Reply:
Since this is SDK for developers I think it makes sense to keep the naming consistent with the API interface and available as a choice so long as SD1 and SD2 are available in the API. Agreed that the naming itself isn't very helpful in knowing what the presets are or how they compare to each other. I believe DreamStudio used FAST_BLUE and just gave a toggle whether to use it or not. It's no longer available in DreamStudio but still available in API.